draft-private-access-tokens.md

```diff
@@ -152,59 +145,125 @@ origins could then track the client's federator user ID or the client's IP
 address across accesses.
 
 
-# Overview
+## Architecture
 
-The architecture and protocol involves the following four entities:
+At a high level, the PAT architecture seeks to solve the following problem: in
+the absence of a stable Client identifier, an Origin needs verification of a
+connecting Client's identity and enforcement of its policy for the incoming
```

Comment:
Here, "its" refers to the Origin, but I'm concerned it could sound like it's for the Client.

```suggestion
the absence of a stable Client identifier, an Origin needs to be able to verify a
connecting Client's identity and enforce access policies for the incoming
```